atchem.f90

```diff
@@ -218,8 +218,8 @@ end subroutine FCVFUN
   call set_model_parameters( getParametersFromFile( trim( param_dir ) //  "/model.parameters" ) )
   write (*,*)
 
-  ! Set the members dayOfYear, dayAsFractionOfYear, secondsInYear of
```

Comment:
I would like to fix everything in one go.  I just realized that theta is not calculated the right way (or rather it is, but if a value for theta is not used for the whole day the equation of time gets screwed up and so does the SZA. I think if theta is corrected then everything else will fall into place.